docs/faq.rst

```diff
@@ -987,6 +987,12 @@ stable. If the directory where you mount a filesystem is different every time,
 Borg assumes they are different files. This is true even if you backup these
 files with relative pathnames - borg uses full pathnames in files cache regardless.
 
+It is possible for some filesystems, such as ``mergerfs`` or network-mounted
```

Comment:
maybe rather say "network filesystems". the mount is on a local directory, but the filesystem is via the network.